app/main.py

```diff
@@ -2,0 +1,20 @@
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for person in friends:
+        try:
+            cafe.visit_cafe(person)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
Code Style: It's a good practice to provide descriptive error messages. While raising the `VaccineError`, you can add a custom message to make the error more descriptive.